MLendPoint/src/middleware/jsonResponse.js

```diff
@@ -1,15 +1,15 @@
 const path = require("path");
 
 class JsonResponse {
-    response(result, algorithm, searchWord, secondsToString,image,response,pathImage) {
+    response(result, algorithm, searchWord, secondsToString, image, response, pathImage) {
```

Comment:
Parameter searchWord is not used in this function you should remove this parameter